pam/internal/adapter/authmodeselection.go

```diff
@@ -222,6 +222,15 @@ func (m authModeSelectionModel) Update(msg tea.Msg) (authModeSelectionModel, tea
 	return m, cmd
 }
 
+// View renders a text view of the authentication UI.
+func (m authModeSelectionModel) View() string {
+	if !m.Focused() {
+		return ""
+	}
+
+	return m.Model.View()
+}
```

Comment:
The problem would be an invisible view if no view is focused, but that would be just a _proper representation_ of a bug, since in any moment a view should be focused.

The problem we had before was that we may end up showing a model view too early, when the view had not input yet and thus the input events were not delivered.

Maybe now (with the latest "Wait" changes I did) that's even not fully needed, but I prefer that in general the UI only prints something if the UI is ready for accepting events.